common/flatpak-run.c

```diff
@@ -3083,7 +3083,10 @@ flatpak_run_add_environment_args (GPtrArray      *argv_array,
 
           g_mkdir_with_parents (src, 0755);
 
-          add_bind_arg (argv_array, "--bind", src, dest);
+          /* We stick to add_args instead of add_bind_arg because persisted
+           * folders don't need to exist outside the chroot.
+           */
```

Comment:
Personally I would drop this extra line, but the file is not consistent (both approaches are used).
Correct spelling is existence. 
No other commit quality comments.  (Not reviewed technical content) +1.

Reply:
Not sure I understand your comment about correct spelling. Could you please clarify?

Reply:
The correct spelling of "existance" is existence.  Not a big deal, just decided to mention it.

Reply:
ah, you mean in the initial comment? I was searching for the word in the patch and wondering if I was going blind! 😅